src/commonMethods/remoteWebInspector.js

```diff
@@ -14,7 +14,7 @@ const WebSocket = require('ws')
 export class AttachToLogs {
   constructor(cb, hostIP, port) {
     this.cb = cb
-    this.port = port || '9998'
+    this.port = port || '9224'
```

Comment:
I think we should not rely on hardcoding the port anymore, since there are apparently differences between images. If you apply this change the BR and vanilla Yocto images will fail, as the CMAKE default is set to 9998. However RDK images overwrite this to 9224 for other reasons.

Instead you can read the WebKitBrowser configuration from the config API and read the port that is configured on the system. We do the same on ThunderUI.